cmd/acmesolver/go.mod

```diff
@@ -6,18 +6,23 @@ go 1.22.0
 // please place any replace statements here at the top for visibility and add a
 // comment to it as to when it can be removed
 
+// Can be removed once expfmt.FmtText is no longer used by any of the libraries we depend on.
+replace github.com/prometheus/common => github.com/prometheus/common v0.46.0
+
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.18.0
```

Comment:
nitpick (non-blocking): Would it be worth expanding on the comment here? WIthout looking up what this comment means, I have no idea what an `expfmt.FmtText` is or how to check if it's not used in libraries we depend on.

Might it also be worth duplicating the comment onto the client_golang replacement to make it clear that that replacement is also affected by the same issue?